lib/client/browser-utils.js

```diff
@@ -82,7 +84,23 @@ function init ($) {
     closeOpenDraw(function () {
       lastOpenedDrawer = id;
       if (prepare) { prepare(); }
-      $(id).css({display:'block', right: '0'});
+
+      var style = {display:'block', right: '0'};
+
+      var windowWidth = $(window).width();
+      if (windowWidth < SMALL_SCREEN || ($(window).height() < SMALL_SCREEN) && windowWidth < 800) {
+        style.width = windowWidth + 'px';
+        var top = $("#chartContainer").offset().top - 45;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2043510332/issues/source?bid=2452379&fileBranchId=3130850#l93)